tests/ai_safety/trustyai_service/constants.py

```
@@ -13,6 +13,11 @@
     "databaseConfigurations": "db-credentials",
 }
 
+DB_CREDENTIALS_SECRET_NAME: str = "db-credentials"
+DB_NAME: str = "trustyai_db"
+DB_USERNAME: str = "trustyai_user"
+DB_PASSWORD: str = "trustyai_password"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "DB_PASSWORD"

(S105)